Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/core/transaction/DccJWK.kt

```diff
@@ -43,4 +48,18 @@ data class DccJWK(
         @SerializedName("enc")
         ENCRYPTION
     }
+
+    @IgnoredOnParcel
+    private val certificateFactory by lazy {
```

Comment:
maybe use a fun with cert factory as param

Reply:
I think `CertificateFactory` is a singleton, what benefit it would give us to use it as a parameter?
btw I'm using a similar approach as we already have in `DscSignatureValidator`

Reply:
No, it is not a Singleton. Check the implementation. And in DscSignatureValidator there is only one instance created and used for every check. You are creating a new Instance for every DccJWK which gets checked..

Reply:
updated